lld/test/ELF/lto/aarch64_inline.ll

```diff
@@ -0,0 +1,51 @@
+; REQUIRES: aarch64
+;; Test verifies inlining happens cross module when module flags are upgraded
+;; by the thin-lto linker/IRMover.
+;; Regression test for #82763
+
+; RUN: split-file %s %t
```

Comment:
This seems unneeded. You can add ThinLTO tests to llvm/test/ThinLTO

Reply:
I added the test here because I'd use `ld.lld` as it invokes the IRMover differently than `llvm-lto`.
and under ` llvm/test/ThinLTO` `ld.lld` is not available.
llvm-lto merges modules to an empty module, while ld.lld merges modules into the first module.

Reply:
Thanks @MaskRay and @teresajohnson ! 
Moved the test, cheers.